portal-web/test/functional/com/liferay/portalweb/macros/ManagementBar.macro

```diff
@@ -11,7 +11,12 @@ definition {
 	}
 
 	macro clickInfo {
-		Click.clickNoMouseOver(locator1 = "Icon#INFO_DM");
+		if (IsElementPresent(locator1 = "Icon#INFO_DM")) {
+			Click.clickNoMouseOver(locator1 = "Icon#INFO_DM");
+		}
+		else {
```

Comment:
It would be better to add a description above it to explain why we need this since the function on master was changed. Thanks in advance.

Reply:
No need to add the description since this is not a specific macro for DM. Sorry